app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java

```diff
@@ -172,6 +176,31 @@ public static void init(Context context) {
         }
     }
 
+    private static SettingsMeasurement.SharedPreferenceSettingsProvider makeSettingsProvider(final Resources resources) {
+        final String prefKeyDefaultBrowser = resources.getString(R.string.pref_key_default_browser);
+        return new SettingsMeasurement.SharedPreferenceSettingsProvider() {
+            @Override
+            public boolean containsKey(String key) {
+                if (key.equals(prefKeyDefaultBrowser)) {
+                    return true;
+                }
+
+                return super.containsKey(key);
+            }
+
+            @Override
+            public java.lang.Object getValue(String key) {
+                if (key.equals(prefKeyDefaultBrowser)) {
+                    final Context context = TelemetryHolder.get().getConfiguration().getContext();
+                    final Browsers browsers = new Browsers(context, "http://www.mozilla.org");
```

Comment:
nit: Since we re-use this `"http://www.mozilla.org"` incantation, we should probably define a constant, if not define a method that makes this use case more apparent.